entrypoint.sh

```diff
@@ -40,4 +40,4 @@ while read -r line; do
             echo "::add-mask::${aws_session_token}"
         fi
     fi
-done < "$credentials"
+done <"$credentials"
```

Comment:
Was this space causing an issue?

Reply:
Don't think so. It's autoformatted by [shell-format](https://marketplace.visualstudio.com/items?itemName=foxundermoon.shell-format) I guess.